api/types.go

```diff
@@ -38,6 +39,19 @@ type networkCreate struct {
 	Options     map[string]interface{} `json:"options"`
 }
 
+// sandboxCreate is the expected body of the "create sandbox" http request message
+type sandboxCreate struct {
+	ID                string                 `json:"id"`
```

Comment:
not sure if that is expected. I think we should create and manage sandbox ID. We can certainly pass the container-id as part of the create object.